openstack/orchestration/v1/stacks/requests.go

```diff
@@ -133,20 +139,6 @@ type AdoptOpts struct {
 	Name string
 	// (REQUIRED) The timeout for stack creation in minutes.
 	Timeout int
-	// (OPTIONAL; REQUIRED IF Template IS EMPTY) The URL of the template to instantiate.
```

Comment:
I decided not to remove the fields in `AdoptOpts`. It should be changed only if/when the documented api is changed